NotificationService/NotificationHandler/appsettings.json

```diff
@@ -1,71 +1,9 @@
 ﻿{
-  "Logging": {
-    "ApplicationInsights": {
-      "LogLevel": {
-        "Default": "Debug",
-        "Microsoft": "Error"
-      }
-    },
-    "LogLevel": {
-      "Default": "Information",
-      "Microsoft": "Warning",
-      "Microsoft.Hosting.Lifetime": "Information"
-    }
-  },
-  "AllowedHosts": "*",
-  "ApplicationInsights": {
-    "InstrumentationKey": "__InstrumentationKey__",
-    "ComponentId": "__ComponentId__",
-    "ComponentName": "__ComponentName__",
-    "Service": "__Service__",
-    "ServiceLine": "__ServiceLine__",
-    "ServiceOffering": "__ServiceOffering__",
-    "TraceLevel": "Information"
-  },
-  "EnvironmentSetting": {
-    "EnvironmentType": "__EnvironmentType__"
-  },
-  "BearerTokenAuthentication": {
-    "ValidAudiences": "__BearerTokenAuthenticationValidAudiences__",
-    "Authority": "__BearerTokenAuthenticationAuthority__",
-    "Issuer": "__BearerTokenAuthenticationIssuer__"
-  },
-  "UserTokenSetting": {
-    "Authority": "__UserTokenSettingAuthority__",
-    "ClientId": "__UserTokenSettingClientId__"
-  },
-  "MSGraphSetting": {
-    "Authority": "__MSGraphSettingAuthority__",
-    "ClientCredential": "__MSGraphSettingClientSecret__",
-    "ClientId": "__MSGraphSettingClientId__",
-    "GraphResourceId": "__MSGraphSettingResourceId__",
-    "GraphAPIVersion": "__MSGraphSettingAPIVersion__",
-    "SendMailUrl": "__MSGraphSettingSendMailUrl__",
-    "TenantId": "__MSGraphSettingTenantId__",
-    "UserAssertionType": "__MSGraphSettingUserAssertionType__",
-    "EnableBatching": "__MSGraphSettingEnableBatching__",
-    "BatchRequestUrl": "__MSGraphSettingBatchUrl__",
-    "BatchRequestLimit": "__MSGraphSettingBatchRequestLimit__"
-  },
-  "RetrySetting": {
-    "MaxRetries": "__MaxRetries__",
-    "TransientRetryCount": "__TransientRetryCount__"
-  },
+  "KeyVaultConfigRefreshDurationSeconds": "120",
   "KeyVault": {
-    "SecretUri": "__KeyVaultSecretUri__",
-    "RSAKeyUri": "__RSAKeyUri__"
-  },
-  "CosmosDB": {
-    "Uri": "__CosmosDBUri__",
-    "Key": "__CosmosDBKey__",
-    "Database": "__CosmosDBDatabase__",
-    "Container": "__CosmosDBContainer__"
-  },
-  "StorageAccount": {
-    "ConnectionString": "__StorageAccountConnectionString__",
-    "BlobContainerName": "__BlobContainerName__",
-    "MailTemplateTableName": "__MailTemplateTableName__"
+    "SecretUri": "__KeyVaultSecretUri__"
   },
-  "StorageType": "__StorageType__",
-  "NotificationProviderType": "__NotificationProviderType__"
+  "AppConfig": {
+    "ForceRefresh": ".appconfig.featureflag/ForceRefresh"
```

Comment:
why do we need featureflag?

Reply:
removing that as there is a automatic refresh of 30sec